Dockerfile

```diff
@@ -17,7 +16,7 @@ ENV NODE_ENV=production
 
 WORKDIR /srv/app/
 
-COPY --from=development /srv/app/ ./
+COPY . .
```

Comment:
Theoretically, you should do the copy of `package.json` and `yarn.lock` before this line too, as well as the `yarn install` to leverage caching. As we practically only use production, which does a yarn install itself, and not the build step for creating a production image we can leave it as is, I think.
Do you want to do it the proper way though or leave it as is too?